src/pr-info.ts

```diff
@@ -185,13 +183,13 @@ export async function deriveStateForPR(
     getDownloads = getMonthlyDownloadCount,
     now = new Date(),
 ): Promise<BotResult>  {
-    if (prInfo.author == null) return botError(prInfo.number, "PR author does not exist");
+    if (prInfo.author == null) return botError("PR author does not exist");
 
-    if (prInfo.isDraft) return botEnsureRemovedFromProject(prInfo.number, "PR is a draft", true);
-    if (prInfo.state !== PullRequestState.OPEN) return botEnsureRemovedFromProject(prInfo.number, "PR is not active", false);
```

Comment:
Also synchronize `botEnsureRemovedFromProject.isDraft` with `prInfo.isDraft`. Cosmetic change, no change in behavior, but doesn't rely on the call flow :point_up:, and is consistent with `botError.author`.